### PR TITLE
📝 clarify what conditions in automations are

### LINKED ISSUE #3578: Automations UI needs clarification on conditions
The automation GUI says:

"Conditions are an optional part of an automation rule and can be used to prevent an action from happening when triggered. "

This implies conditions will prevent an action happening if a state is evaluated as true.  So if a user was to check for a state that should cancel the action when that state is true - the action would actually continue.

This is incorrect as conditions function  as: all conditions must be true to continue execution.

This could probably do with clarification that:

"Conditions can prevent an automation from continuing, all conditions must be true to continue execution."

This is in the GUI here:

https://i.imgur.com/QoRPjKc.jpg

### PR DESCRIPTION
**Pull request by iantrich**
fixes https://github.com/home-assistant/home-assistant-polymer/issues/3578

### CODE REVIEW

File: src/translations/en.json [hunk 1]
```diff
@@ -895,7 +895,7 @@
             "conditions": {
               "name": "Condition",
               "header": "Conditions",
-              "introduction": "Conditions are an optional part of an automation rule and can be used to prevent an action from happening when triggered. Conditions look very similar to triggers but are very different. A trigger will look at events happening in the system while a condition only looks at how the system looks right now. A trigger can observe that a switch is being turned on. A condition can only see if a switch is currently on or off.",
+              "introduction": "Conditions are an optional part of an automation rule and can be used to prevent an action from happening if not all conditions are true when triggered. Conditions look very similar to triggers but are very different. A trigger will look at events happening in the system while a condition only looks at how the system looks right now. A trigger can observe that a switch is being turned on. A condition can only see if a switch is currently on or off.",
```
Comment (balloob) [inline src/translations/en.json]: Maybe better?

```suggestion
              "introduction": "Conditions are an optional part of an automation rule and will prevent further execution of a triggered automation unless all conditions are satisfied. Conditions look very similar to triggers but are very different. A trigger will look at events happening in the system while a condition only looks at how the system looks right now. A trigger can observe that a switch is being turned on. A condition can only see if a switch is currently on or off.",
```

**Comment by balloob** [comment]
I've been thinking about this more. Do we really need this? It has worked just fine. Any text we put in will at some point confuse some user. However, if the sentences get too long or complicated, users might end up not reading it at all and we are working against ourselves.

**Comment by SeanPM5** [comment]
I feel the current text is already too long and should be shortened to just the first sentence. There doesn't need to be a whole paragraph explaining a fairly simple concept, especially given the fact that it's completely optional. 

By the second sentence it already begins to get overly confusing for beginners. When reading that, they will be thinking: it's "very similar" but also "very different" to this "triggers" thing I just learned about two seconds ago? Huh?

Keep it short and simple imo, there's a "learn more" link to the docs underneath if the user wants to know every aspect of how it works.

**Comment by iantrich** [comment]
@balloob @SeanPM5 thanks for the feedback. Made it short and sweet and allow for complete explanation in the docs link

**Comment by balloob** [comment]
I like it. Agree with Sean.

**Comment by Aterfax** [comment]
I'd probably amend "conditions satisfied" to be "conditions true" just to leave no doubt.

**Pull request merged**